tests/query/parser/test_query.py

```diff
@@ -64,13 +63,30 @@ def with_required(condition: Optional[Expression] = None) -> Expression:
     return required
 
 
+DEFAULT_TEST_QUERY_CONDITIONS = [
+    "timestamp >= toDateTime('2021-01-01T00:00:00')",
```

Comment:
We could extract out more aspects of the datetime and use the constants in the assertions above in `with_required`. But I don't know if we get much benefit from that